spark-connect/index.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: global
+type: "page singular"
+title: Spark Connect
+description: Spark Connect makes remote Spark development easier.
+subproject: Spark Connect
+---
+
+This post explains the Spark Connect architecture, the benefits of Spark Connect, and how to upgrade to Spark Connect.
```

Comment:
`This post` sounds a little weird in Apache Spark website. Is this copied from your blog?

Reply:
No, I didn't copy this from anywhere, but happy to reword it.  Any suggestions?  Perhaps "This page" would be better.